### PR TITLE
Continuous Integration Adjustments

### PR DESCRIPTION
**Pull request by ziadoz**
Some adjustments here:

- Use Ubuntu + PHP rather than Docker in CI builds.
- Support PHP 7.4 and 8.0.
- Improve Bash alias code.

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 name: "Continuous Integration"
 
 on:
-  - workflow_dispatch
-  - pull_request
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
```
Comment (maxakropolis) [inline .github/workflows/continuous-integration.yml:10]: Why only run on master?

Comment (ziadoz) [reply .github/workflows/continuous-integration.yml:10]: @maxakropolis This means the CI workflow will only run on pull requests against the master base branch.

File: .github/workflows/continuous-integration.yml [hunk 2]
```diff
@@ -1,62 +1,51 @@
 name: "Continuous Integration"
 
 on:
-  workflow_dispatch:
+  push:
```
Comment (maxakropolis) [inline .github/workflows/continuous-integration.yml:5]: Do you need to include `workflow_dispatch:`?

Comment (ziadoz) [reply .github/workflows/continuous-integration.yml:5]: @maxakropolis I have. I added it back at the top.

**Review by maxakropolis — APPROVED** [review]

**Pull request merged**